### ISSUE TITLE
ICE with vs2019 16.7.2

### ISSUE THREAD

**Issue by mihaisebea** [microsoft/vcperf#12]
```
vcperf\src\timetrace\executionhierarchy.cpp : fatal error C1001: Internal compiler error.
1>(compiler file 'd:\agent\_work\8\s\src\vctools\Compiler\Utc\src\p2\main.c', line 195)
1> To work around this problem, try simplifying or changing the program near the locations listed above.
1>If possible please provide a repro here: https://developercommunity.visualstudio.com
1>Please choose the Technical Support command on the Visual C++
1> Help menu, or open the Technical Support help file for more information
1>  link!RaiseException()+0x62
1>  link!CxxThrowException()+0x66
1>  link!std::_Xout_of_range()+0x1f
1>  link!InvokeCompilerPass()+0x432b5
1>  link!InvokeCompilerPass()+0x3016a
1>  link!InvokeCompilerPass()+0x312b4
1>  link!InvokeCompilerPass()+0x412c6
1>  link!InvokeCompilerPassW()+0x100fd9
```

**Comment by mihaisebea** [comment]
16.8.0 preview 2 seems to work btw

**Comment by kevcadieux** [comment]
I can't repro this with latest MSVC. You've also indicated that it was fixed in 16.8. I will close this issue. Thanks!
